source/styleEditor.js

```diff
@@ -1,3 +1,12 @@
 $(document).ready(function(){
-  // your code goes here.
+  $('#style_editor').submit(function() {
+  event.preventDefault();
```

Comment:
This might just be a github thing, but make sure to indent the body of a function